test/cctest/test_url.cc

```diff
@@ -109,3 +109,36 @@ TEST_F(URLTest, ToFilePath) {
 
 #undef T
 }
+
+TEST_F(URLTest, FromFilePath) {
+  URL file_url;
+#ifdef _WIN32
+  file_url = URL::FromFilePath("C:\\Program Files\\");
+  EXPECT_EQ(file_url.protocol(), "file:");
+  EXPECT_EQ(file_url.path(), "/C:/Program%20Files/");
+
+  file_url = URL::FromFilePath("C:\\a\\b\\c");
+  EXPECT_EQ(file_url.protocol(), "file:");
+  EXPECT_EQ(file_url.path(), "/a/b/c");
```

Comment:
Hmm, these look wrong. It should probably be `file:///C:/a/b/c`. See failing CI run.

Reply:
done.